src/pages/home/index.tsx

```diff
@@ -7,10 +7,23 @@ import HomeChatListByTag from '@/components/home/HomeChatListByTag';
 import * as Styled from './home.styled';
 import RecommendedUsers from '@/components/@common/Recommendations/User/RecommendedUsers';
 import HomeFriendListByTag from '@/components/home/HomeFriendListByTag';
+import Banner from '@/components/home/Banner';
 
 const HomePage = () => {
   return (
-    <>
+    <Styled.Wrapper>
+      {/*배너 영역 */}
+      <Banner
+        roomId={5}
+        title={
+          <Styled.BannerTitle>
+            <h3>화제의 기대작!</h3>
```

Comment:
```suggestion
            <h3>
            화제의 기대작!<br />
            오징어게임 진심인<br />
            사람들과 이모저모 수다 떨기
            </h3>
```

br 태그를 사용하면 h3 태그를 한 번만 사용할 수 있을 것 같습니다☺️

Reply:
수정했습니다!